packages/react-native/android/src/main/java/com/dashx/sdk/DashXData.kt

```diff
@@ -20,3 +20,15 @@ data class TrackRequest(
 data class TrackResponse(
     val success: Boolean
 )
+
+data class SubscribeRequest(
+    val deviceToken: String,
+    val deviceName: String,
+    val deviceKind: String,
+    val uid: String?,
+    val anonymous_uid: String?
```

Comment:
There seems to be some inconsistency with the casing.

Reply:
Renaming